### PR TITLE
Update cni binary

### LINKED ISSUE #23011: NETWORK_PROVIDER=kubenet broken on GCE
Event for a pod: 
```console
Error syncing pod, skipping: failed to "SetupNetwork" for "kube-dns-v11-zbufd_kube-system" with SetupNetworkError: "Failed to setup network for pod \"kube-dns-v11-zbufd_kube-system(ea05b8bd-eada-11e5-8ced-42010af00002)\" using network plugins \"kubenet\": Error adding container to network: fork/exec /opt/cni/bin/bridge: no such file or directory; Skipping pod"
```

The cni plugins do exist at /opt/cni/bin. They cannot be executed and bash is complaining:
```console
root@e2e-test-mixia-minion-suyg:/opt/cni/bin# ls -al
total 37964
drwxr-x--- 2 8119 5000    4096 Jan 28 21:17 .
drwxr-xr-x 3 root root    4096 Mar 15 18:22 ..
-rwxr-xr-x 1 8119 5000 4781424 Jan 28 21:17 bridge
-rwxr-x--- 1 8119 5000 3615096 Jan 28 21:17 cnitool
-rwxr-x--- 1 8119 5000 9233472 Jan 28 21:17 dhcp
-rwxr-x--- 1 8119 5000 4017040 Jan 28 21:17 flannel
-rwxr-x--- 1 8119 5000 3863224 Jan 28 21:17 host-local
-rwxr-x--- 1 8119 5000 4256816 Jan 28 21:17 ipvlan
-rwxr-x--- 1 8119 5000 4256872 Jan 28 21:17 macvlan
-rwxr-x--- 1 8119 5000 4821840 Jan 28 21:17 ptp
root@e2e-test-mixia-minion-suyg:/opt/cni/bin# ./bridge 
bash: ./bridge: No such file or directory
```

### PR DESCRIPTION
**Pull request by freehan**
fixes #23011

### CODE REVIEW

File: pkg/kubelet/network/kubenet/kubenet_linux.go [hunk 1]
```diff
@@ -139,6 +139,7 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 		cidr.IP.To4()[3] += 1
 
 		json := fmt.Sprintf(NET_CONFIG_TEMPLATE, BridgeName, plugin.MTU, network.DefaultInterfaceName, podCIDR, cidr.IP.String())
+		glog.V(2).Infof("CNI network config set to %v", json)
```
Comment (bprashanth) [inline pkg/kubelet/network/kubenet/kubenet_linux.go]: Do you want to print out the entire json blob at V(2) on every event? 
I guess right now there's only 1 event, sugges printing the event type at V(2) and the whole json at V(4).

**Comment by bprashanth** [comment]
LGTM but for the logging nit. Marking this lgtm so if it goes in, it goes in and you can send a follow up pr (today printing out a relatively small amount of json at V(2) once isn't too bad given how much the kubelet logs, but it will be bad when we grow number of Events/json).

File: pkg/kubelet/network/kubenet/kubenet_linux.go (hunk 1 above)
Comment (freehan) [inline pkg/kubelet/network/kubenet/kubenet_linux.go]: Typically, this log only output once when PodCIDR is set though. It will not happened repeatedly. See line #120 of this file. And it makes folks life much easier to debug CNI stuff, given it is already tough.

Comment (bprashanth) [inline pkg/kubelet/network/kubenet/kubenet_linux.go]: This happens on every Event, it so happens that the only event today is pod cidr. Upto you we can always dumb it down later. Unsure if a customer is going to care about the details of pod networking, though.

Comment (freehan) [inline pkg/kubelet/network/kubenet/kubenet_linux.go]: Check this: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/kubenet/kubenet_linux.go#L122
It only process podCIDR change event. This is not a good code structure, tough. But we can fix it later.

Comment (bprashanth) [inline pkg/kubelet/network/kubenet/kubenet_linux.go]: Hah. Suggest either renaming the function, or allowing generic events. And logging when a !POD_CIDR_CHANGE is observed at Warningf.

**Comment by freehan** [comment]
hand merging since #23712 is depending on this
